### DISCUSSION TITLE
Curso: Lógica de Programação com Dart

### DISCUSSION THREAD

**Discussion by everaldojunior95** [balta-io/forum#367] [Tira Dúvidas] [answered]
Olá professor!! 
Sou totalmente novo no assunto de programação, não tenho conhecimento sobre as plataformas e decidi dá um primeiro passo fazendo o curso de Lógica de Programação com vcs. Estou com dificuldade para seguir conforme a sequência da aula. Na situação em questão, o professor pede para que eu "dê um Dart . V" no "Terminal". Como / onde eu acesso esse "terminal"? É no GitHub? Como procedo? Aguardo uma orientação.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @everaldojunior95, como vai?

Caso esteja utilizando o Windows, recomendo usar o Windows Terminal (Normalmente já vem instalado).
No Mac ou Linux chama-se Terminal!

Qual sistema operacional está utilizando?

**Comment by dezin2001** [comment]
Professor @andrebaltieri ,também sou totalmente novo no ramo. Queria tirar uma dúvida com você, o conteúdo desse curso que eu irei aprender, irei conseguir desenvolver em qualquer plataforma que use linguagem DART?

**Reply by andrebaltieri** [reply]
Bom dia, @dezin2001, sim! O Dart é a linguagem do Flutter, que é um Framework multi-plataforma, ou seja, suporta Android, iOS, Windows, Mac e Linux.
